### ISSUE TITLE
Kubernetes Aggregated Failures is stuck

### ISSUE THREAD

**Issue by dims** [kubernetes/test-infra#4381]
Looking at https://storage.googleapis.com/k8s-gubernator/triage/index.html 

```
1298 clusters of 222282 failures (25776 in last day) out of 106349 builds from 8/24/2017, 5:13:41 PM to 9/1/2017, 6:22:26 PM.
```

Looks like it keeled over on 9/1 and never recovered. Can someone please nudge it?

**Comment by rmmh** [comment]
Yes, I'm working on it.

**Comment by rmmh** [comment]
Fixed by #4426
